core/poi/lib/plugins/config-css.js

```diff
@@ -6,7 +6,9 @@ exports.cli = ({ command, isProd }) => {
   if (isProd) {
     command.option('--no-extract-css', `Don't extract CSS files`)
   } else {
-    command.option('--extract-css', 'Extract CSS files')
+    command.option('--extract-css', 'Extract CSS files', {
+      default: true
```

Comment:
for some reason we can't set default value for cli flags, instead we do it here: https://github.com/egoist/poi/blob/e4b5b04a3f0c4c278fd4b277e609bb52423056bf/core/poi/lib/utils/validateConfig.js#L77